<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>656</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="uiNameLine">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Activity name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="uiSetButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="uiIDLabel">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Value:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QDoubleSpinBox" name="uiValueSpin">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string extracomment="value"/>
      </property>
      <property name="minimum">
       <double>-9999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="uiUnitLine">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Unit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="5">
     <widget class="QPushButton" name="uiSubmitButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Submit activity</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="4">
     <widget class="QLineEdit" name="uiSpeciesLine">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Species</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>uiNameLine</tabstop>
  <tabstop>uiSetButton</tabstop>
  <tabstop>uiValueSpin</tabstop>
  <tabstop>uiUnitLine</tabstop>
  <tabstop>uiSpeciesLine</tabstop>
  <tabstop>uiSubmitButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uiSubmitButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>submit_activity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiSetButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>set_activity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_activity()</slot>
  <slot>submit_activity()</slot>
 </slots>
</ui>
